src/goto-analyzer/unreachable_instructions.cpp

```diff
@@ -74,6 +85,33 @@ static void output_dead_plain(
     goto_program.output_instruction(ns, "", os, *it->second);
 }
 
+static void add_to_xml(
+  const namespacet &ns,
+  const goto_programt &goto_program,
+  const dead_mapt &dead_map,
+  xmlt &dest)
+{
+  assert(!goto_program.instructions.empty());
+  goto_programt::const_targett end_function=
+    goto_program.instructions.end();
+  --end_function;
+  assert(end_function->is_end_function());
+
+  xmlt &x = dest.new_element("function");
+  x.set_attribute("name", id2string(end_function->function));
+
+  for(dead_mapt::const_iterator it=dead_map.begin();
+      it!=dead_map.end();
+      ++it)
+  {
+    xmlt &inst = x.new_element("instruction");
+    inst.set_attribute("location_number",
+                       std::to_string(it->second->location_number));
+    inst.set_attribute("source_location", it->second->source_location.as_string());
+  }
+  return;
```

Comment:
Unnecessary

Reply:
I tend to put them in to distinguish between "yes, that's it, we are done here" and "I got distracted and forgot to write the rest".  Not that that tends to...

*squirrel*

Huh?

Reply:
I see your point, but I'm not sure that's done consistently? As you know, I don't deal well with surprises.